elastic/kibana
# [Uptime] Fix sending flag to indicate edits when updating service monitors

## Summary

This PR fixes Kibana not sending the `is_edit` flag for when monitors are updated. That's necessary so that the service PR for https://github.com/elastic/synthetics-service/issues/244 can work.

It seems like this bug was introduced in https://github.com/elastic/kibana/pull/131270 by removing the extra argument passed to `pushConfigs`.

## How to test this PR

> ⚠️ Please make sure to also check testing instructions in the service PR: https://github.com/elastic/synthetics-service/pull/505

1. Checkout the appropriate Kibana version (this PR) and start it pointing to the service (make sure to checkout the branch with the "rerun on edit" changes)
    ```yaml
    # Use the URL of the service here
    xpack.uptime.service.devUrl: https://localhost:10001
    ```
2. Checkout the appropriate version of the service, build it, and start it 
3. Create a monitor in Kibana
    ```js
    step("visit google", async () => {
        await page.goto("https://google.com");
    });
    ```
4. Go to Uptime and check the time this monitor has last run
5. Edit the monitor's code
    ```js
    step("visit HackerNews", async () => {
        await page.goto("https://news.ycombinator.com");
    });
    ```
6. Once you've saved it, give it a few seconds to re-run and check the `Uptime` page. You should soon get results for the edited monitor with the new URL.


### Checklist

Delete any items that are not applicable to this PR.



### For maintainers
